mesh/mesh.hpp

```diff
@@ -206,12 +206,25 @@ class Mesh
    void ReadLineMesh(std::istream &input);
    void ReadNetgen2DMesh(std::istream &input, int &curved);
    void ReadNetgen3DMesh(std::istream &input);
+   void ReadNetgenSurfaceMesh(std::istream &input);
+   void ReadNetgenVol(std::istream &input);
    void ReadTrueGridMesh(std::istream &input);
    void ReadVTKMesh(std::istream &input, int &curved, int &read_gf,
                     bool &finalize_topo);
    void ReadNURBSMesh(std::istream &input, int &curved, int &read_gf);
    void ReadInlineMesh(std::istream &input, int generate_edges = 0);
+
+   void ReadGmshV2(std::istream &input, int binary);
+   void ReadGmshV4(std::istream &input, int binary);
+   void ReadGmshV41(std::istream &input, int binary);
```

Comment:
I don't think this is something to tackle now, but I'm starting to think we will want to want to pull stuff like this into some kind of mesh reader classes.

Reply:
Agree.